### PR TITLE
Not compatible with PHP 5.x

### LINKED ISSUE #1: Not compatible with PHP 5.x
Hi, I'm using composer-gavel as a globally-installed composer plugin, so that I'm always reminded to set the composer version constraint on projects. However, the plugin requires PHP 7, but some of my projects are still stuck back on PHP 5.6. So I get this error on running composer commands:

```
Parse error: syntax error, unexpected '::' (T_PAAMAYIM_NEKUDOTAYIM) in /Users/xyz/.composer/vendor/deviantintegral/composer-gavel/src/ComposerVersionRequirement.php on line 54
```

Could the code be tweaked to be backwards-compatible with PHP 5.x?

Or could there be a way to avoid the code running at all on certain projects? I've googled around and found that globally-installed plugins could still read settings from the specific project, to disable themselves for that project, for example. See https://github.com/fxpio/composer-asset-plugin/commit/fda53b9edbe53621d55e05f29a8e9b1095e32079#diff-43154fc4cec0f5b9a65d66620720dd28L104

I would guess that since it's a parse error, disabling at that point within the class wouldn't work as the class still has to be parsed, but maybe the class could be wrapped... but then maybe it's easier/better to just have backwards-compatible code?

### PR DESCRIPTION
**Pull request by anotherjames**
These are suggested changes that I believe would resolve #1 .

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
         }
     ],
     "require": {
-        "php": ">=7.0.0",
+        "php": ">=5.3.0",
```
Comment (deviantintegral) [inline composer.json]: This should be at least PHP 5.6, as all prior versions are EOL'ed.

**Comment by anotherjames** [comment]
Done ... but I didn't know much about CircleCI before. I see an older version of PHPUnit would be needed (v5), although that itself is unsupported now too. Would you be OK with allowing PHPUnit 5? (I use this plugin with Drupal projects, and Drupal's requirement is `>=4.8.28 <5`!)

**Comment by anotherjames** [comment]
@deviantintegral I've no idea what that test failure is about? Sorry!

**Comment by deviantintegral** [comment]
Yes, using the older PHPUnit version is fine.

I think the build failed on the last step that submits code coverage reports to Code Climate. I've turned on passing environment variables to forked PRs so let's see if that works.

**Comment by anotherjames** [comment]
Wow, thanks for following up on this!

**Pull request merged**